community/cypher/docs/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/MatchTest.scala

```diff
@@ -203,16 +203,18 @@ This is not recommended practice. See <<match-node-by-id>> for more information
   }
 
   @Test def relationshipsByTypeWithSpace() {
-    db.inTx {
-      val a = node("Rob")
-      val b = node("Charlie")
-      a.createRelationshipTo(b, DynamicRelationshipType.withName("TYPE THAT HAS SPACE IN IT"))
-    }
-    testQuery(
+//    db.inTx {
+//      val a = node("Rob")
+//      val b = node("Charlie")
+//      a.createRelationshipTo(b, DynamicRelationshipType.withName("TYPE THAT HAS SPACE IN IT"))
+//    }
+
+    prepareAndTestQuery(
```

Comment:
Oh, I forgot to remove the comment. Will fix. The actual change is the addition of line 217.